tests/models/test_hooks.py

```diff
@@ -329,16 +335,19 @@ def on_test_model_train(self):
         'on_before_zero_grad',
         'on_train_batch_end',
         'on_validation_model_eval',
+        'on_validation_start',
         'on_validation_epoch_start',
         'on_validation_batch_start',
         'on_validation_batch_end',
         'on_validation_epoch_end',
         'on_save_checkpoint',
+        'on_validation_end',
         'on_validation_model_train',
         'on_epoch_end',
         'on_train_epoch_end',
         'on_train_end',
         'on_fit_end',
+        'teardown',
     ]
 
     assert model.called == expected
```

Comment:
shall this be updated the way it's done with callback using mock?

Reply:
that would be great (in a follow up pr). let me know if you need help

Reply:
sure :+1: